typings/manifold.bundle.d.ts

```diff
@@ -1,4 +1,4 @@
-// manifesto.js v1.0.0 https://github.com/viewdir/manifesto
+// manifesto.js v2.0.1 https://github.com/viewdir/manifesto
```

Comment:
I'd support that decision. Actually in my latest pushes I removed TreeNode usage for straight Manifest uses.